File: Services/Models/Order.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Services.Models
{
   public class Order
    {
        public int OrderId { get; set; }
        public string Comment { get; set; }
        public string Address { get; set; }
        public DateTime OrderDate { get; set; }
        public DateTime ShipmentTime { get; set; }
        public int Price { get; set; }
        //public User UserBuyer { get; set; }
        public int UserBuyerId { get; set; }
        public List<OrderProduct> OrderProducts { get; set; }
    }
}

File: web2-front-app/src/Models/Order.js
import Product from "./Product";

export class Order {
    constructor(
      OrderId=0,
      Comment="",
      Address="",
      OrderDate=null,
      Products=[],
      UserBuyerId=0
      
    ) {
      this.orderId=OrderId;
      this.comment=Comment;
      this.address=Address;
      this.orderDate=OrderDate;
      this.products=Products;
      this.userBuyerId=UserBuyerId;
    }
  }
  export default Order;
  
  